doc/about.rst

```diff
@@ -176,14 +176,14 @@ The 2013' Paris international sprint
    :target: http://www.telecom-paristech.fr/
 
 
-.. |tinyclues| image:: http://www.tinyclues.com/static/img/logo.png
+.. |tinyclues| image:: https://www.tinyclues.com/static/img/logo.png
```

Comment:
Right, I think this should work? https://www.tinyclues.com/web/wp-content/uploads/2016/06/Tinyclues-PNG-logo.png  The link from their homepage doesn't show an image for me:
https://www.tinyclues.com/web/wp-content/themes/tinyclues/img/tinyclues-logo.svg